examples/atomic-registry/Makefile

```diff
@@ -21,7 +21,7 @@ install-atomic:
 install-systemd:
 	atomic install $(TEST_IMAGE) $(INSTALLHOST)
 	systemctl enable --now atomic-registry-master.service
-	sleep 10
+	sleep 30
```

Comment:
If you're doing a `docker run` you can marry a `ExecStartPre="docker run -d container"` with `ExecStart="docker logs -f container"`. Or you may be able to do a `docker inspect` with a template to wait until something specific in the Docker API. See discussion [here](https://github.com/docker/docker/issues/6791#issuecomment-72338100) -- bumping a sleep seems like a poor way out, but I'll defer to you if you think it's out of scope for this PR.